src/bindings/python/src/pyopenvino/frontend/extensions.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+#include "common/extensions/decoder_transformation_extension.hpp"
+#include "common/extensions/telemetry_extension.hpp"
+#include "common/frontend_exceptions.hpp"
+#include "json_extension/json_config_extension.hpp"
+#include "manager.hpp"
+#include "pyopenvino/graph/function.hpp"
+
+namespace py = pybind11;
+
+using namespace ov::frontend;
+
+void regclass_frontend_Extension(py::module m) {
+    py::class_<ov::Extension, std::shared_ptr<ov::Extension>> ext(m, "Extension", py::dynamic_attr());
```

Comment:
Why do you register ov::Extension as frontend Extension?

Reply:
fixed